_assets/styles/scss/04-components/_forms--comment.scss

```diff
@@ -154,3 +147,105 @@ parent: forms
     border-width: 3px;
   }
 }
+
+// The entire text editor component.
+.text-editor {
+  background: $white;
+  border: 1px solid $form-element-border;
+  border-radius: 3px;
+  font-family: $post-font;
+  padding: 1em;
+
+  &:hover {
+    border-color: $form-element-border-hover;
+  }
+}
+
+// Style buttons.
+.text-editor__controls {
+  font-size: .9em;
+  margin-bottom: .25em;
+  user-select: none;
+}
+
+.text-editor__style-button {
+  color: darken($light-grey, 25%);
+  cursor: pointer;
+  display: inline-block;
+  margin-right: 1em;
+  padding: 2px 0;
+}
+
+.text-editor__style-button--active {
+  color: $orange;
+}
+
+// The text area within the editor.
+.text-editor__editor {
+  border-top: 1px solid $form-element-border;
+  cursor: text;
+  font-size: 1em;
+  margin-top: .5em;
+
+  .public-DraftEditorPlaceholder-root,
+  .public-DraftEditor-content {
+    margin: 0 -1em -1em;
+    padding: 1em;
+  }
+
+  .public-DraftEditor-content {
+    min-height: 100px;
+  }
+
+  h3,
+  h4 {
+    font-weight: 400;
+    margin: .75em 0 .5em;
+
+    &:first-child {
+      margin-top: 0;
+    }
+  }
+}
+
+// Code blocks.
+.public-DraftStyleDefault-pre {
+  background-color: $light-gray;
```

Comment:
I might be missing something but it looks like we use both `grey` and `gray` throughout the SCSS files